StripePaymentSheet/StripePaymentSheet/Source/Helpers/SavedPaymentMethodManager.swift

```diff
@@ -6,26 +6,52 @@
 //
 
 import Foundation
+@_spi(STP) import StripeCore
 @_spi(STP) import StripePayments
 
 /// Provides shared implementations of common operations for managing saved payment methods in PaymentSheet
 final class SavedPaymentMethodManager {
 
+    enum Error: Swift.Error {
+        case missingEphemeralKey
+    }
+
     let configuration: PaymentSheet.Configuration
+    let intent: Intent
 
-    init(configuration: PaymentSheet.Configuration) {
```

Comment:
I considered making this init throws if we can't find a valid eph key, but didn't love the ergonomics of it.